data/data/bootstrap/gcp/files/usr/local/bin/report-progress.sh

```diff
@@ -9,12 +9,15 @@ wait_for_existance() {
 	done
 }
 
-echo "Waiting for bootstrap to complete..."
-wait_for_existance /opt/openshift/.bootkube.done
+echo "Waiting for cb-bootstrap to complete..."
+wait_for_existance /opt/openshift/cb-bootstrap.done
```

Comment:
Should this be a hidden file?

Reply:
Yeah good eye the general standard for the other containers is to not use dotfile [1] but then `.bootkube.done` does. :man_shrugging: So I think we are still ok.

https://github.com/openshift/installer/blob/43134e2de6dc1936f769ef95617ebc7e5cb1b5ca/data/data/bootstrap/files/usr/local/bin/bootkube.sh.template#L401

Reply:
https://github.com/openshift/installer/blob/43134e2de6dc1936f769ef95617ebc7e5cb1b5ca/data/data/bootstrap/files/usr/local/bin/bootkube.sh.template#L263